dependencies.gradle

```diff
@@ -65,34 +65,46 @@ ext {
-            work          : "android.arch.work:work-runtime:$workVersion",
-            lottie        : "com.airbnb.android:lottie:$lottieVersion",
-            debugDb       : "com.amitshekhar.android:debug-db:$debugDbVersion",
-            skeleton      : "com.ethanhua:skeleton:$skeletonVersion",
-            shimmer       : "io.supercharge:shimmerlayout:$shimmerVersion"
+            kotlin            : "org.jetbrains.kotlin:kotlin-stdlib-jdk7:$kotlinVersion",
+            appCompat         : "androidx.appcompat:appcompat:$appCompatVersion",
+            material          : "com.google.android.material:material:$materialVersion",
+            koin              : "org.koin:koin-android:$koinVersion",
+            koinViewModel     : "org.koin:koin-android-viewmodel:$koinVersion",
+            timber            : "com.jakewharton.timber:timber:$timberVersion",
+            navFragment       : "android.arch.navigation:navigation-fragment-ktx:$navigationVersion",
+            navUi             : "android.arch.navigation:navigation-ui-ktx:$navigationVersion",
+            fragment          : "androidx.fragment:fragment-ktx:$fragmentVersion",
+            rxAndroid         : "io.reactivex.rxjava2:rxandroid:$rxAndroidVersion",
+            rxKotlin          : "io.reactivex.rxjava2:rxkotlin:$rxKotlinVersion",
+            archRuntime       : "androidx.arch.core:core-runtime:$lifecycleVersion",
+            archExtension     : "androidx.lifecycle:lifecycle-extensions:$lifecycleVersion",
+            archCompiler      : "androidx.lifecycle:lifecycle-compiler:$lifecycleVersion",
+            constraint        : "androidx.constraintlayout:constraintlayout:$constraintVersion",
+            ktx               : "androidx.core:core-ktx:$ktxVersion",
+            glide             : "com.github.bumptech.glide:glide:$glideVersion",
+            glideCompiler     : "com.github.bumptech.glide:compiler:$glideVersion",
+            epoxy             : "com.airbnb.android:epoxy:$epoxyVersion",
+            epoxyProcessor    : "com.airbnb.android:epoxy-processor:$epoxyVersion",
+            work              : "android.arch.work:work-runtime:$workVersion",
+            lottie            : "com.airbnb.android:lottie:$lottieVersion",
+            debugDb           : "com.amitshekhar.android:debug-db:$debugDbVersion",
+            skeleton          : "com.ethanhua:skeleton:$skeletonVersion",
+            shimmer           : "io.supercharge:shimmerlayout:$shimmerVersion",
+            gson              : "com.google.code.gson:gson:${gsonVersion}",
+            androidAnnotations: "com.android.support:support-annotations:${androidAnnotationsVersion}",
+            okHttp            : "com.squareup.okhttp3:okhttp:${okHttpVersion}",
+            okHttpLogger      : "com.squareup.okhttp3:logging-interceptor:${okHttpVersion}",
+            retrofit          : "com.squareup.retrofit2:retrofit:${retrofitVersion}",
```

Comment:
**NIT** would be better if we follow similar conventions for using variables throughout the file. `${someVersion}` or `$someVersion`

Reply:
Done